tests/system/test_system.py

```diff
@@ -1803,11 +1814,12 @@ def test_bucket_w_default_event_based_hold(self):
         self.assertFalse(bucket.retention_policy_locked)
 
         blob.upload_from_string(payload)
-        self.assertFalse(other.event_based_hold)
-        self.assertFalse(other.temporary_hold)
-        self.assertIsNone(other.retention_expiration_time)
+        self.assertFalse(blob.event_based_hold)
+        self.assertFalse(blob.temporary_hold)
+        self.assertIsNone(blob.retention_expiration_time)
```

Comment:
I think, there is an error here: `other` blob is already deleted (line 1806). Most likely, `blob` was going to be here, as it's just has been uploaded.

Reply:
Good catch